plugins/modules/nxos_pim_interface.py

```diff
@@ -511,13 +517,16 @@ def config_pim_interface_defaults(existing, jp_bidir, isauth):
     return command
 
 
-def normalize_proposed_values(proposed):
+def normalize_proposed_values(proposed, module):
     keys = proposed.keys()
     if "bfd" in keys:
         # bfd is a tri-state string: enable, disable, default
         proposed["bfd"] = proposed["bfd"].lower()
     if "hello_interval" in keys:
-        proposed["hello_interval"] = str(proposed["hello_interval"] * 1000)
+        hello_interval = proposed["hello_interval"]
+        if not module.params["hello_interval_ms"]:
```

Comment:
if "hello_interval_ms" is absent in module.params this might throw keyerror.is this required attribute?

Reply:
The keys are always populated in the module.params. If they are not explicitly specified in task, then they are set to None. There is a test case that covers this scenario in this PR https://github.com/ansible-collections/cisco.nxos/pull/255/files#diff-5f307170102de468b4ba6dda3fb5c0bb398e5c6f0fefc66c8ad8d9a6a66b7055R296